### ISSUE TITLE
vars-on-top conflicts with ES6 `import`

### ISSUE THREAD

**Issue by srigi** [eslint/eslint#2099]
When `vars-on-top` is enabled, it warns (IMO incorrectly) on this ES6 code

```js
import React from 'react';

var foo = 'bar';
```

It wants `var` declaration to be before `import` call. That is unnecessary.

I'm using `babel-eslint` parser and `browser` env.

**Comment by nzakas** [comment]
Yeah, I think the rule just doesn't understand import. Should be an easy fix.

**Comment by gyandeeps** [comment]
Working on this.
@nzakas What should be allowed for this rule to not fail? 
`import` for sure, anyothers

**Comment by nzakas** [comment]
I think starting with `import` is good. Maybe there will be more, but I'm not sure at this point.
